docs/api/lights/shadows/LightShadow.html

```diff
@@ -134,6 +134,11 @@ <h3>[method:Object toJSON]()</h3>
 		Serialize this LightShadow.
 		</p>
 
+		<h3>[method:void update]( [param:Light light] )</h3>
+		<p>
+		Updates this LightShadow (e.g. [page:SpotLightShadow.update SpotLightShadow.update]).
```

Comment:
This method is akin to a virtual base class and does not do anything. Define what it does where it is overloaded.